internal/cluster/distribution/eks/eksworkflow/activity_update_addon_test.go

```diff
@@ -61,26 +61,82 @@ func TestSelectLatestVersion(t *testing.T) {
 							},
 						},
 					},
+					{
+						AddonVersion: aws.String("v1.8.5-eksbuild.1"),
+						Compatibilities: []*eks.Compatibility{
+							{
+								ClusterVersion: aws.String("1.20"),
+							},
+						},
+					},
+					{
+						AddonVersion: aws.String("v1.8.8-eksbuild.1"),
+						Compatibilities: []*eks.Compatibility{
+							{
+								ClusterVersion: aws.String("1.20"),
+							},
+						},
+					},
+					{
+						AddonVersion: aws.String("v1.9.0-eksbuild.1"),
+						Compatibilities: []*eks.Compatibility{
+							{
+								ClusterVersion: aws.String("1.20"),
+							},
+						},
+					},
+					{
+						AddonVersion: aws.String("v1.9.1-eksbuild.1"),
+						Compatibilities: []*eks.Compatibility{
+							{
+								ClusterVersion: aws.String("1.20"),
+							},
+						},
+					},
```

Comment:
**Question**: shouldn't these be part of the `coredns` addon's `AddonVersions`? Or if we deliberately add a new one, at least we should give it a name as well regardless of what it is (the input is not used).

If you opt for merging, the 1.8.5 version should be merged with the existing 1.8.5 verison with 2 compatibilities entry for k8s 1.19 and 1.20.

Reply:
fixed